.github/workflows/build-manylinux-container-images.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           YEAR: _2_31  # There are no base images prior to 2.31 for this arch
 
     env:
-      LIBSSH_VERSION: 0.11.2
+      LIBSSH_VERSION: 0.11.3
```

Comment:
This shouldn't be changed in this PR. It'll only confuse you into thinking that it has any effect on your CI run but it doesn't. You'll just achieve triggering more pointless jobs while working on the actual change.

Reply:
I dropped this commit which I pulled in by accident during rebase.